lib/jsdom/living/nodes/Element-impl.js

```diff
@@ -430,6 +430,43 @@ class ElementImpl extends NodeImpl {
 
     return attr;
   }
+
+  insertAdjacentHTML(position, text) {
+    const parentNode = this.parentNode;
+    const container = this.ownerDocument.createElementNS("http://www.w3.org/1999/xhtml", "_");
```

Comment:
Hmm reading the spec it seems it does something a bit more complicated, e.g. defaulting to body in certain cases. I think it's OK to just use template as it will be most correct, but be careful and check browsers. If there are some edge cases that neither "template" nor "body" cover, then it's OK, we can just leave a TODO.